Sources/Scheduler.swift

```diff
@@ -264,16 +264,16 @@ public final class QueueScheduler: DateSchedulerProtocol {
 		precondition(interval >= 0)
 		precondition(leeway >= 0)
 
-		let nsecInterval = interval * Double(NSEC_PER_SEC)
-		let nsecLeeway = leeway * Double(NSEC_PER_SEC)
+		let msecInterval = interval * 1000
+		let msecLeeway = leeway * 1000
 
 		let timer = DispatchSource.makeTimerSource(
 			flags: DispatchSource.TimerFlags(rawValue: UInt(0)),
 			queue: queue
 		)
 		timer.scheduleRepeating(wallDeadline: wallTime(with: date),
-		                        interval: .nanoseconds(Int(nsecInterval)),
-		                        leeway: .nanoseconds(Int(nsecLeeway)))
+		                        interval: .milliseconds(Int(msecInterval)),
```

Comment:
Oh I forgot the parameter is a `TimeInterval`. Okay, then I think the right fix, which is not backwards compatible, but definitely best long term, is to change the parameter to take a `DispatchTimeInterval`. It's actually a much better API because it self-documents the magnitude. With these old APIs taking `Double`s it's never clear whether it's seconds or what.